apps/docs/pages/guides/auth/passwords.mdx

```diff
@@ -0,0 +1,155 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+import { Accordion } from 'ui'
+
+export const meta = {
+  id: 'auth-passwords',
+  title: 'Passwords',
+  description: 'How to work with passwords in Supabase Auth',
+}
+
+Using passwords to authenticate users is a tried-and-tested method to give your users access to your application. Supabase Auth provides you with secure configuration options and uses best-practices to store and verify your user's passwords.
+
+## Password security
+
+Passwords derive their security from how easy they are to guess or brute-force In theory this means that the longer a password is and the more required characters it has (digits, lowercase and uppercase letters, symbols) the harder it will be to guess.
+
+This table shows the _minimum_ number of guesses that need to be tried to access a user's account:
+
+| Required characters                          | Length | Guesses          |
+| -------------------------------------------- | ------ | ---------------- |
+| Digits only                                  | 8      | ~ 2<sup>27</sup> |
+| Digits and letters                           | 8      | ~ 2<sup>41</sup> |
+| Digits, lower and uppercase letters          | 8      | ~ 2<sup>48</sup> |
+| Digits, lower and uppercase letters, symbols | 8      | ~ 2<sup>52</sup> |
+
+In reality though, passwords are not always generated at random, but often contain variations of names, words, dates, sentences or phrases. Malicious actors can use these properties to guess a password in less attempts.
+
+There are hundreds of millions (and growing!) known passwords out there. Malicious actors can use these lists of leaked passwords to automate login attempts (known as credential stuffing) and steal or access sensitive user data.
```

Comment:
Also, passwords are often reused. Malicious actors can use lists of leaked passwords to automate login attempts (known as credential stuffing).

Reply:
Would actually prefer to bring up the leaked passwords first.